service/history/timer_queue_active_task_executor_test.go

```diff
@@ -1965,7 +1965,7 @@ func (s *timerQueueActiveTaskExecutorSuite) getMutableStateFromCache(
 		WorkflowKey: definition.NewWorkflowKey(namespaceID.String(), workflowID, runID),
 		ShardUUID:   s.mockShard.GetOwner(),
 	}
-	return s.workflowCache.(*wcache.CacheImpl).Get(key).(*workflow.ContextImpl).MutableState
```

Comment:
This makes me sad. But I can't fix the leaky design as part of this PR. So I had to add a helper function that allows to still access the MutableState directly here. :-/